app/parsers/pbcore_xml_parser.rb

```diff
@@ -149,7 +149,10 @@ def set_objects(file, index)
 
     pi_rows = pbcore_physical_instantiations.map do |inst|
       xml_pi = AAPB::BatchIngest::PBCoreXMLMapper.new(inst.to_xml).physical_instantiation_attributes
-      physical_instantiation = PhysicalInstantiation.where(local_instantiation_identifier: xml_pi[:local_instantiation_identifier]).first&.attributes&.symbolize_keys
+      # Find members of the asset that have the same class and local identifier. If no asset, then no physical instantiation can exist
+      physical_instantiation = asset.members.detect do |member|
+          member.local_instantiation_identifier == xml_pi[:local_instantiation_identifier] && member.class == PhysicalInstantiation
+        end if asset && xml_pi[:local_instantiation_identifier].present?
```

Comment:
unindent i think?

Reply:
yep. I'll merge and then handle so we don't have to re-review